examples/webgpu_cubemap_adjustments.html

```diff
@@ -72,6 +72,12 @@
 				cube2Texture.generateMipmaps = true;
 				cube2Texture.minFilter = THREE.LinearMipmapLinearFilter;
 
+				renderer = new THREE.WebGPURenderer( { antialias: true } );
+				await renderer.init();
```

Comment:
This was a quick fix to deal with an "uninitialized renderer" warning, which should maybe be an error since it just skips my shader entirely and makes a black texture? It also says I should use `renderAsync`, but i'm using `camera.update`, which doesn't seem to have an async version? Not sure why this hasn't been a problem for `fromEquirectangularTexture`...